arbiter/arbiter-core/src/main/java/org/deeplearning4j/arbiter/optimize/generator/genetic/Chromosome.java

```diff
@@ -0,0 +1,39 @@
+/*******************************************************************************
+ * Copyright (c) 2015-2018 Skymind, Inc.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ******************************************************************************/
+
+package org.deeplearning4j.arbiter.optimize.generator.genetic;
+
+/**
+ * Candidates are stored as Chromosome in the population model
+ *
+ * @author Alexandre Boulanger
+ */
+public class Chromosome {
+    /**
+     * The fitness score of the genes.
+     */
+    public final double fitness;
```

Comment:
Let's use private or protected here... that goes for all other public fields too (I won't point out every instance in this review).
And add a lombok ```@Data``` annotation to the class, to add getters, toString, equals etc
In case you aren't familiar with lombok: https://projectlombok.org/ - you'll also need IDE plugin.